src/assets/scss/layout/layout.scss

```diff
@@ -0,0 +1,43 @@
+@import "../../../assets/scss/helpers/variables";
+
+.pf-c-dropdown {
+  &__menu {
+    &-item {
+      color: #000 !important;
+    }
+  }
+}
+
+.footer {
+  display: flex;
+  position: fixed;
+  height: 55px;
+  bottom: 0px;
+  left: 0px;
+  right: 0px;
+  width: 100%;
+
+  .footer-text {
+    color: white;
+    text-align: center;
+  }
+  
+  .footer-link {
+    display: flex;
+    align-items: center;
+    
+  .footer-logo {
+    width: 50px;
+    margin-left: 10px;
+    margin-right: 10px;
+    }
```

Comment:
I might be wrong but it looks like that bracket is not needed there.

Reply:
that bracket is needed for footer-link

footer-logo is nested inside footer-link., so there has to be 2 sets of braces for them

Reply:
Makes sense, can you please properly indent `.footer-logo` so it looks like it's actually nested? Thanks

Reply:
Done!